src/engine/renderer/tr_image.cpp

```diff
@@ -315,8 +357,26 @@ void R_ListImages_f()
 		}
 		else
 		{
-			format = imageFormatNameSize.at( image->internalFormat ).first;
-			imageDataSize *= imageFormatNameSize.at( image->internalFormat ).second;
+			switch ( image->internalFormat ) {
+				// Compressed formats encode blocks of 4x4 texels
+				case GL_COMPRESSED_RGBA:
+				case GL_COMPRESSED_RED_RGTC1:
+				case GL_COMPRESSED_SIGNED_RED_RGTC1:
+				case GL_COMPRESSED_RG_RGTC2:
+				case GL_COMPRESSED_SIGNED_RG_RGTC2:
+				case GL_COMPRESSED_RGB_S3TC_DXT1_EXT:
+				case GL_COMPRESSED_RGBA_S3TC_DXT1_EXT:
+				case GL_COMPRESSED_RGBA_S3TC_DXT3_EXT:
+				case GL_COMPRESSED_RGBA_S3TC_DXT5_EXT:
+					format = imageFormatNameSize.at( image->internalFormat ).first;
+					imageDataSize = ceil( image->uploadWidth / 4.0 ) * ceil( image->uploadHeight / 4.0 ) * image->numLayers
+									* imageFormatNameSize.at( image->internalFormat ).second;
```

Comment:
Better do a single tab character for a single indentation, ascii art is not for indentation.

Reply:
You mean for line 373? I do that to have formulas left-aligned, but I can change it to a single tab.

Reply:
Yes, the line 373. We better avoid to abuse ascii art and reserve it for some schematics in comments or so.

Continuing a line on the next line is better considered an indentation as convention.

In case it would be very wanted (I don't think of any use case anyway), in-block alignment should be done with space as tab size is variable.

Basically:

- indentation = tabs only
- ascii art = spaces only

Doing that makes the spacing being semantic for the human reader, and avoids all the spacing mismatches.

Reply:
That's true w. r. t. tabs yeah.

Reply:
Changed this to 1 tab now.